.env.example

```
@@ -0,0 +1,2 @@
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/panda-market
```

env example에는 명확한 값을 안 넣어주는 것이 좋습니다
```suggestion
DATABASE_URL=postgresql://postgres:postgres@localhost:5432/database
```

---

env example은 미션3 모범답안에 포함되어있으며 실제로 사용되지 않는 것으로 알고 있습니다. 실제 db명과도 차이가 있는데 혹시 문제가 될 수 있을까요?